### PR TITLE
Add org.kde.ktrip

### PR DESCRIPTION
**Pull request by nicolasfella**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
I am the upstream maintainer
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nicolasfella** [comment]
bot, build org.kde.ktrip

File: org.kde.ktrip.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.kde.ktrip",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "ktrip",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.ktrip.json:9]:
```suggestion
        "--socket=fallback-x11",
```

File: org.kde.ktrip.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.04.1/src/kpublictransport-21.04.1.tar.xz",
+                    "sha1": "e863af0df7ac2c2aec1c512853d3d8b2cd8b6ba1"
+                }
+            ]
+        },
+        {
+            "name": "kirigami-addons",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "archive", "url": "https://download.kde.org/unstable/kirigami-addons/0.2/kirigami-addons-0.2.tar.xz", "sha1": "a9c5c913fd7cbb1ebe17bffef32a1c1d744a9a33"} ]
```
Comment (hfiguiere) [inline org.kde.ktrip.json:30]: sha256 are preferred.

File: org.kde.ktrip.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        {
+            "name": "ktrip",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "archive", "url": "https://download.kde.org/stable/plasma-mobile/21.05/ktrip-21.05.tar.xz", "sha1": "25a1d6ad85b3454ea42b2aa43fe0c8b8bb9810e9" }
```
Comment (hfiguiere) [inline org.kde.ktrip.json:36]: sha256 preferred

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by nicolasfella** [comment]
The appstream issues will be fixed with the 21.06 release in a few days

**Comment by nicolasfella** [comment]
bot, build org.kde.ktrip

**Comment by nicolasfella** [comment]
bot, build org.kde.ktrip

**Comment by barthalion** [comment]
cc @flathub/kde

File: org.kde.ktrip.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+
+    "modules": [
+        {
+            "name": "kpublictransport",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.04.1/src/kpublictransport-21.04.1.tar.xz",
```
Comment (tsdgeos) [inline org.kde.ktrip.json:22]: .2 was relased today ^_^

**Comment by nicolasfella** [comment]
bot, build org.kde.ktrip

**Pull request closed without merging**